app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Admin::UsersController < ApplicationController
```

Comment:
@iori19960828 Please user `CategoriesController` to display category related data.